python/python/bystro/ancestry/tests/test_ancestry_types.py

```diff
@@ -0,0 +1,276 @@
+"""Test ancestry_types.py."""
+import pytest
+from attrs.exceptions import FrozenInstanceError
+
+from ..ancestry_types import (
+    AncestryResponse,
+    AncestryResult,
+    AncestrySubmission,
+    AttrValidationError,
+    PopulationVector,
+    ProbabilityInterval,
+    SuperpopVector,
+)
+
```

Comment:
Could you add a brief overview of how these types are being tested at the beginning? I saw a little description under some of them but just to get a quick idea could be useful, or if you wanted to instead add a little description to every subtype for more clarity.

Reply:
I added some more expansive docstrings and renamed some of the tests for clarity.  Going to land this now so we can keep the ball rolling but pls lmk if you'd like additional docs